internal/test_cases/await_command_test_case.go

```
@@ -0,0 +1,84 @@
+	}
+
+	t.ReceivedResponse = value
+
+	<-t.AwaitChan
+
+	if err = t.Assertion.Run(value); err != nil {
+		return err
+	}
+
+	if !t.ShouldSkipUnreadDataCheck {
+		client.ReadIntoBuffer() // Let's make sure there's no extra data
```

golangci-lint (1.64.8)

Error return value of `client.ReadIntoBuffer` is not checked

(errcheck)